### ISSUE TITLE
Cost Dashboard Instructions

### ISSUE THREAD

**Issue by haorasu** [aws-samples/aws-batch-operational-dashboards#17]
The instructions for the Cost Dashboard is slightly off as there are no data sources in the json file that references Cloudwatch.

This is what I see: 
![Screenshot 2023-11-08 at 12 17 38 PM](https://github.com/aws-samples/aws-batch-operational-dashboards/assets/140550341/8448e11c-fbb2-4ad0-a2f1-6435e4436846)

**Comment by haorasu** [comment]
Potentially related issue Post deployment of the above:

Error updating options: error querying the database: TABLE_NOT_FOUND: line 1:40: Table 'aws-batch-jobs-data.default.awsbatchcostreport' does not exist

**Comment by mhuguesaws** [comment]
Will address before end of year.

**Comment by mhuguesaws** [comment]
Looking at end of Q1 2024.

**Comment by iMilad** [comment]
any update on this? I get the same data sources and issue as OP mentioned.

**Comment by mhuguesaws** [comment]
Updated. Let me know if anything else is missing.

**Comment by iMilad** [comment]
now I have to check why athena does not have enough permission
`Status: 500. Message: error executing query: InvalidRequestException: Unable to verify/create output bucket batch-op-dashboard-athenaspills3bucket-8kxc { RespMetadata: { StatusCode: 400, RequestID: "6e4cdf1d-d310299f18dd" }, AthenaErrorCode: "INSUFFICIENT_PERMISSIONS", Message_: "Unable to verify/create output bucket batch-op-dashboard-athenaspills3bucket-8kxc" }`

**Comment by iMilad** [comment]
now I get this error, `Status: 500. Message: COLUMN_NOT_FOUND: line 1:40: Column 'resource_tags_user_project' cannot be resolved or requester is not authorized to access requested resources` any hint?
checked the `athenacurcfn_a_w_s_batch_cost_report.awsbatchcostreport` and there is not such a column
![image](https://github.com/aws-samples/aws-batch-operational-dashboards/assets/9392720/0db2968d-9b4b-40cb-a4c3-7ce21a0ee3be)

**Comment by mhuguesaws** [comment]
It will take 24 hours to populate the CUR as per documentation.

**Comment by iMilad** [comment]
any hint?

**Comment by iMilad** [comment]
how this tag "resource_tags_user_project" will be generated? or which service should have this tag? Batch job?
we have Batch which is used by different team and we want to understand cost of batch per team.

**Comment by mhuguesaws** [comment]
This a tag in the batch job

Tag Name: project
Tag Value: XXXX

**Comment by iMilad** [comment]
No, this is not working, I added the tag as you mentioned to the job but still after two days there is no such a column in awsbatchcostreport.

![image](https://github.com/aws-samples/aws-batch-operational-dashboards/assets/9392720/fbb61efb-b1d9-45e6-9d48-0b75ef1a0bc8)

**Comment by cyberchip-wang** [comment]
Have you activated the tag under [Billing and Cost Management](https://us-east-1.console.aws.amazon.com/costmanagement/home#/home)
->Cost Allocation Tags after that?
![image](https://github.com/aws-samples/aws-batch-operational-dashboards/assets/111794603/b875ea7f-ec2d-4245-bb14-de5df85a5986)

After activation, you should see the column in CUR in another day or so.

**Comment by iMilad** [comment]
thank you, I think in the end it worth to mention in the README.

**Comment by mhuguesaws** [comment]
@iMilad did it solve your issue?

**Comment by iMilad** [comment]
yes, this issue is resolved. thanks!

**Comment by mhuguesaws** [comment]
Will add to the doc.
